docs/versioned_docs/version-v1.0.0-beta.0/getting_started/quick_start.md

```diff
@@ -13,7 +13,7 @@ The easiest way to develop with Noir is using Nargo the CLI tool. It provides yo
 You can use `noirup` the installation script to quickly install and update Nargo:
 
 ```bash
-curl -L https://raw.githubusercontent.com/noir-lang/noirup/refs/heads/main/install | bash
+curl -L noirup.dev | bash
```

Comment:
We stopped using these urls right?

Reply:
I wasn't aware of that, they're fixed now though. This PR is just me going back one commit behind the release tag and copying the docs folder to versioned_docs as the beta.0 version

Happy to discuss stuff but maybe as part of a different PR?